www/_blog/2021-11-28-postgrest-9.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: 'PostgREST 9'
+description: New features and updates in PostgREST version 9.
+author: steve_chavez
+author_url: https://github.com/steve-chavez
+author_image_url: https://github.com/steve-chavez.png
+authorURL: https://github.com/steve-chavez
+image: launch-week-three/community-day/supabase-oss.jpg
```

Comment:
@thorwebdev just re-tagging this one as I don't think it's resolved?

Reply:
@kiwicopple sorry, got it locally. just ticking off so I know what's left to do. Will push in a sec.

Reply:
I know, not the best workflow 🙈

Reply:
all good!